lib/internal/bootstrap/realm.js

```diff
@@ -426,7 +426,9 @@ function requireBuiltin(id) {
 // Allow internal modules from dependencies to require
 // other modules from dependencies by providing fallbacks.
 function requireWithFallbackInDeps(request) {
-  if (!BuiltinModule.map.has(request)) {
+  if (StringPrototypeStartsWith(request, 'node:')) {
+    request = BuiltinModule.normalizeRequirableId(request);
```

Comment:
`normalizeRequirableId` has a lot of unnecessary checks (e.g. `canBeRequiredByUsers`), I wonder if it wouldn't be simpler to just slice.
```suggestion
    request = StringPrototypeSlice(request, 5);
```

Reply:
I actually did this at first but then reps would technically be able to require things that they typically can't. Probably not a problem though I just wasn't sure what was better

Reply:
done!